python/veles/db.py

```diff
@@ -448,9 +455,10 @@ def set_bindata(self, id, key, start, data, truncate=False):
         ])
 
         # We're done here.
-        self.db.commit()
+        if commit:
+            self.commit()
 
-    def delete(self, id):
+    def delete(self, id, commit=True):
```

Comment:
Hmm, maybe we can also use it in `AsyncLocalNode.delete` to only commit one delete instead of commit for every child of deleted Node?